### ISSUE TITLE
NameError: name 'BaseSecurityManager' is not defined

### ISSUE THREAD

**Issue by heyarnold1** [dpgaspar/Flask-AppBuilder#1849]
### Environment

Flask-Appbuilder version: 4.0.1

### Describe the expected results

On run the application should startup with no errors

### Describe the actual results

we get the following error

```pytb
  File "/Users/henry/nook/app/backend/.venv/lib/python3.9/site-packages/flask_appbuilder/base.py", line 147, in __init__
    self.init_app(app, session)
  File "/Users/henry/nook/app/backend/.venv/lib/python3.9/site-packages/flask_appbuilder/base.py", line 199, in init_app
    Type[BaseSecurityManager], security_manager_class
NameError: name 'BaseSecurityManager' is not defined
```

as TYPE_CHECKING is always false it is unclear to me how this would not always fail

### Steps to reproduce

On starting up our app

```python
adminapp = Flask(__name__)
adminapp.config.from_object("admin.config")
db = SQLA(adminapp)
appbuilder = AppBuilder(adminapp, db.session)```

**Comment by dpgaspar** [comment]
Thank you for reporting this issue a fix is merged to master and we will release soon

**Comment by dpgaspar** [comment]
fix release on 4.1.1
